### ISSUE TITLE
plotQC dgCMatrix coercion to data.frame error

### ISSUE THREAD

**Issue by danfulop** [alanocallaghan/scater#41]
`plotQC()` threw this error on a 10X dataset:

```
> plotQC(all.data)
Error in as.data.frame.default(x[[i]], optional = TRUE) : 
  cannot coerce class "structure("dgCMatrix", package = "Matrix")" to a data.frame
```

**Comment by LTLA** [comment]
This is also an issue with the not-yet-updated `plotHighestExprs` function in 1.6.1, see #38.

@davismcc?

**Comment by LTLA** [comment]
This should be fixed in #39, though I don't think this has made it to release.
